x/profile/keeper/msg_server_update_coordinator_description.go

```diff
@@ -30,6 +30,13 @@ func (k msgServer) UpdateCoordinatorDescription(
 				coordByAddress.CoordinatorID)
 	}
 
+	// Check if the coordinator is inactive
+	if !coord.Active {
```

Comment:
Same as above, we can use `GetActiveCoordinatorByAddress`

Reply:
I got some confusion between `getCoordinatorByAddress` and `GetCoordinatorByAddress`
Yes, this is a good point. Although, I believe having an extra read in the store is not a big concern if it's related to a single operation, as long as it doesn't become more than O(n) complexity

So we have `GetCoordinatorByAddress` used by external module where the invariant is checked and internal method using `getCoordinatorByAddress` where the invariant is checked internally because `Coordinator` mst be fetched as well

Reply:
We can keep this behaviour although in any case, we must return a Critical error if an inactive coordinator has been fetched with `getCoordinatorByAddress`, so in this particular condition, a critical error must be returned